### PR TITLE
Setup logging

### PR DESCRIPTION
**Pull request by lgonczar**

### CODE REVIEW

File: environment_acceptance.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 export SETTINGS='config.TestConfig'
 export PPI_END_POINT='http://landregistry.local:8002/get_ppi'
-export ELASTIC_SEARCH_ENDPOINT='http://ec2-54-72-123-155.eu-west-1.compute.amazonaws.com::4300'
+export ELASTIC_SEARCH_ENDPOINT='http://ec2-54-77-238-128.eu-west-1.compute.amazonaws.com:4300'
```
Comment (camillebaldock) [inline environment_acceptance.sh]: why are we changing back the elastic search endpoint here? :) believe we want to stay on 123 155, but without two "dots" in the URL where the port is? (cc @nickgravgaard is that the bug you were hunting?)

File: property-api-log-config.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  },
+  "root": {
+    "level": "INFO",
+    "handlers": [
+      "file",
+      "console"
+    ]
+  }
+}
```
Comment (camillebaldock) [inline property-api-log-config.json]: newline at end of file missing here :)

File: environment_acceptance.sh (hunk 1 above)
Comment (lgonczar) [inline environment_acceptance.sh]: Right, a merge issue

File: property-api-log-config.json (hunk 2 above)
Comment (lgonczar) [inline property-api-log-config.json]: done

**Comment by camillebaldock** [comment]
LGTM!

**Pull request merged**